src/Gir.Tests/InterfaceTests.cs

```diff
@@ -157,7 +157,7 @@ public interface IComponent
 		/// a handler id which can be used in atk_component_remove_focus_handler()
 		/// or zero if the handler was already added.
 		///</returns>
-		uint AddFocusHandler PARAMS
```

Comment:
So, if a parent of a method is an interface, we need to generate the function definition without the boxy. `public` modifier is extraneous. We also should not generate pinvoke imports.